src/test/java/com/urswolfer/gerrit/client/rest/http/changes/ChangesRestClientTest.java

```diff
@@ -54,16 +54,16 @@
         return Iterables.transform(Arrays.asList(
                 queryParameter(
                         new TestQueryRequest().withQuery("is:open")
-                ).expectUrl("/changes/?q=is:open"),
+                ).expectUrl("/changes/?q=is%3Aopen"),
```

Comment:
Not sure what you mean here, but I guess it's basically the same question as the one about backwards compatibility?